sdk/textanalytics/azure-ai-textanalytics/swagger/README.md

```diff
@@ -0,0 +1,90 @@
+# Text Analytics for Python
+
+To generate this file, simply type
+
+```
+autorest samples/README.md
```

Comment:
why the readme of the swagger is in the samples folder?

Reply:
oh thanks, mistyped that, should be swagger